src/service/filterService.js

```diff
@@ -0,0 +1,28 @@
+/**
+ *  filterService - provides service methods for filters like channel filters
+ *
+ * author: Loganathan Shanmugam
+ * email: loganathan.shanmugam@tarento.com
+ */
+var whiteListedChannelList = process.env.sunbird_content_service_whitelisted_channels
+var blackListedChannelList = process.env.sunbird_content_service_blacklisted_channels
+
+function getChannelSearchString (req, callback) {
+  var searchString = generateChannelSearchString()
+  callback(null, searchString)
+}
+
+function generateChannelSearchString () {
```

Comment:
dont need to do this on every API call. this can be processed and cached on server startup.